sr_description/test/test_hands.sh

```diff
@@ -3,7 +3,24 @@
 
 source ./test_models.sh
 
-launch sr_hand hand_type:="hand_e" hand_version:="E3M5" side:="right" fingers:="th,ff,lf" tip_sensors:="pst"
-launch sr_hand hand_type:="hand_e" hand_version:="E3M5" side:="left" fingers:="th,ff,lf" tip_sensors:="pst"
-launch sr_hand hand_type:="hand_e" hand_version:="E3M5" side:="right" fingers:="th,ff,lf" tip_sensors:="th=bt_2p,ff=bt_sp,mf=pst,rf=pst,lf=pst"
+# Testing default hand - hand_type:="hand_e" hand_version:="E3M5" side:="right" fingers:="all" tip_sensors:="pst"
 launch sr_hand
+
+# Testing left side
+launch sr_hand side:="left"
```

Comment:
instead of having to kill every step with ctrl+c would be nice to just start a process and kill it with enter button. E.g. like here:
https://stackoverflow.com/questions/22867130/bash-script-to-start-process-wait-random-kill-process-restart/22867414